.gitignore

```diff
@@ -40,7 +40,7 @@ crash.log
 # Ignore any .tfvars files that are generated automatically for each Terraform run. Most
 # .tfvars files are managed as part of configuration and so should be included in
 # version control.
-**/*.tfvars
+#**/*.tfvars
```

Comment:
We should keep `tfvars` files ignored. For the files you have as part of the environment/dev folder: let's either directly set those values in the modules, or rename to `terraform.tfvars.example` so it doesn't get picked up by the filter

Reply:
I agree, I already made the change.